designer/server/src/common/components/tag-env/template.njk

```diff
@@ -0,0 +1,15 @@
+{%- from "govuk/components/tag/macro.njk" import govukTag -%}
+
+{%- switch env %}
+    {% case "test" %}
+      {% set classes = "govuk-tag--yellow" %}
+    {% case "production" %}
+      {% set classes = "govuk-tag--red" %}
+    {% default %}
+      {% set classes = "govuk-tag--grey" %}
```

Comment:
Sorry @KiranVarma02 these inner lines still need indenting

```suggestion
  {% case "test" %}
    {% set classes = "govuk-tag--yellow" %}
  {% case "production" %}
    {% set classes = "govuk-tag--red" %}
  {% default %}
    {% set classes = "govuk-tag--grey" %}
```

Reply:
Done. If I am working on this project for longer, I would add lint rule for this. 

Maybe I will look for it this afternoon.